scripts/install_astyle.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+rm -rf astyle
+git clone https://github.com/lvgl/astyle.git
```

Comment:
Is the lvgl/astyle.git going to be frozen to a particular commit or is it better to checkout a particular version here?

Reply:
`install_astyle.sh` takes care of installing the correct version by git tag.